x-pack/plugins/reporting/server/core.ts

```diff
@@ -144,6 +150,8 @@ export class ReportingCore {
     this.pluginSetup$.next(true); // trigger the observer
     this.pluginSetupDeps = setupDeps; // cache
 
+    this.pdfExport.setup(this.core, setupDeps);
```

Comment:
It looks like we don't really need the first parameter to `setup`. The core object is already given in the pdfExport's constructor.